querydsl-root/pom.xml

```diff
@@ -285,12 +293,16 @@
               <packages>com.querydsl.spatial*</packages>
             </group>
             <group>
-              <title>Lucene</title>
-              <packages>com.querydsl.lucene3:com.querydsl.lucene4</packages>
+              <title>Lucene 3</title>
+              <packages>com.querydsl.lucene3</packages>
+            </group>
+            <group>
+              <title>Lucene 4</title>
+              <packages>com.querydsl.lucene4</packages>
             </group>
             <group>
               <title>Hibernate Search</title>
-              <packages>com.querydsl.search</packages>
+              <packages>com.querydsl.hibernate.search</packages>
```

Comment:
Shouldn't these all have a wildcard?
I admit, there are no more packages, but for consistency (and errors?) it might be better to add them.
What do you think?